yt_dlp/extractor/instagram.py

```diff
@@ -42,13 +43,15 @@ class InstagramBaseIE(InfoExtractor):
 
     _API_BASE_URL = 'https://i.instagram.com/api/v1'
     _LOGIN_URL = 'https://www.instagram.com/accounts/login'
-    _API_HEADERS = {
-        'X-IG-App-ID': '936619743392459',
-        'X-ASBD-ID': '198387',
-        'X-IG-WWW-Claim': '0',
-        'Origin': 'https://www.instagram.com',
-        'Accept': '*/*',
-    }
+
+    def _api_headers(self):
```

Comment:
I'd prefer to do this, and then remove all the `()`s below
```suggestion
    @property
    def _api_headers(self):
```